### PR TITLE
Sorry for last error :)

### PR DESCRIPTION
**Pull request by FiloSpaTeam**
I changed some things. 
* Now there is a real ToolBar more adaptable to GNU/Linux DE and Mac OSx.
* Update IT translations
* Some refactor about QtPassSettings Singleton (i missed some instances :grin: )
* Now Git Pull and Git Push Button are always visible - as suggested from GNOME 3 Guidelines, users prefers to have buttons visible and disable than hide/show.

### CODE REVIEW

**Comment by annejan** [comment]
No worries, that's why we have automated testing 😁

**Comment by FiloSpaTeam** [comment]
Oh, if you prefer to have TextBesideIcon change the code. Actually i set to FollowStyle (ToolBar icons)

**Comment by annejan** [comment]
I think I prefer FollowStyle since it's more coherent.

**Comment by FiloSpaTeam** [comment]
Ok, perfect. Another thing. Why Travis CI build failed? I can't understand :laughing:

**Comment by annejan** [comment]
Restarted the job, Travis has glitches sometimes . .
Looks like it timed-out installing a prerequisites.

File: src/qtpasssettings.cpp [hunk 1]
```diff
@@ -129,23 +133,23 @@ void QtPassSettings::setSize(const QSize &size) {
...
+//}
+
+// int QtPassSettings::getSplitterRight(const int &defaultValue) {
+//  return getInstance()
+//      ->value(SettingsConstants::splitterRight, defaultValue)
+//      .toInt();
+//}
+// void QtPassSettings::setSplitterRight(const int &splitterRight) {
+//  getInstance()->setValue(SettingsConstants::splitterRight, splitterRight);
+//}
```
Comment (annejan) [inline src/qtpasssettings.cpp]: If you could remove these (instead of commenting out) that would be wonderful.

**Comment by FiloSpaTeam** [comment]
Ok understood! Thanks.
If you want some changes tell me :+1:

File: src/qtpasssettings.h [hunk 2]
```diff
@@ -57,11 +57,11 @@ class QtPassSettings : public QSettings {
...
 
-  static int getSplitterLeft(const int &defaultValue = QVariant().toInt());
-  static void setSplitterLeft(const int &splitterLeft);
+  //  static int getSplitterLeft(const int &defaultValue = QVariant().toInt());
+  //  static void setSplitterLeft(const int &splitterLeft);
 
-  static int getSplitterRight(const int &defaultValue = QVariant().toInt());
-  static void setSplitterRight(const int &splitterRight);
+  //  static int getSplitterRight(const int &defaultValue = QVariant().toInt());
+  //  static void setSplitterRight(const int &splitterRight);
```
Comment (annejan) [inline src/qtpasssettings.h]: Same with these commented out lines.

**Pull request merged**